composer.json

```diff
@@ -32,13 +32,19 @@
       "url": "https://wpackagist.org"
     }
   ],
+  "autoload": {
+    "psr-4": {
+      "Roots\\Bedrock\\": "lib/Bedrock"
+    }
+  },
   "require": {
     "php": ">=5.6",
     "composer/installers": "^1.4",
     "vlucas/phpdotenv": "^2.0.1",
     "johnpbloch/wordpress": "4.9.8",
     "oscarotero/env": "^1.1.0",
-    "roots/wp-password-bcrypt": "1.0.0"
+    "roots/wp-password-bcrypt": "1.0.0",
+    "roots/wp-config": "dev-master"
```

Comment:
+1 for extracting this as a package!

How about `^1.0` instead of `dev-master`?

Reply:
Yep! I am going tag 1.0.0 before this is merged